provisionerd/proto/version.go

```diff
@@ -50,6 +50,9 @@ import "github.com/coder/coder/v2/apiversion"
 //
 // API v1.9:
 //   - Added new field named 'has_external_agent' in 'CompleteJob.TemplateImport'
+//
+// API v1.10:
+//   - Added new field `tooltip` in `App`
 const (
 	CurrentMajor = 1
 	CurrentMinor = 9
```

Comment:
you'll need to bump this

```suggestion
	CurrentMinor = 10
```

Reply:
Good catch! Fixed in https://github.com/coder/coder/pull/19651/commits/c2d6191352f62bd369de723115ed3dacf1db49b3